buildSrc/src/main/java/Versions.kt

```diff
@@ -20,14 +20,14 @@ object Versions {
     const val constraintLayout = "2.1.1"
     const val materialDesign = "1.4.0"
     const val lifeCycle = "2.5.1"
-    const val compose_compiler_ext = "1.2.0"
-    const val compose_material3 = "1.0.1"
-    const val compose_version = "1.2.0-rc03"
-    const val compose_activity_version = "1.6.0-alpha05"
+    const val compose_compiler_ext = "1.5.3"
+    const val compose_material3 = "1.1.1"
```

Comment:
Can we upgrade it to latest stable version, [1.1.2](https://androidx.tech/artifacts/compose.material3/material3/1.1.2)?